components/table_kv/src/memory.rs

```diff
@@ -25,6 +25,16 @@ pub enum Error {
 
     #[snafu(display("Primary key duplicate.\nBacktrace:\n{}", backtrace))]
     PrimaryKeyDuplicate { backtrace: Backtrace },
+
+    #[snafu(display(
+        "Table delete data error, table:{}.\nBacktrace:\n{}",
+        table_name,
+        backtrace
+    ))]
+    DeleteData {
```

Comment:
Is this error used?

Reply:
Yes, it is used when failed to delete data.